content/_index.md

```diff
@@ -24,18 +24,29 @@ In short, it would let anyone easily answer the question, "does this product con
 {{< /home/section >}}
 
 {{< home/section title="How?" >}}
+**How does [GitBOM](glossary/gitbom/) improve software identification and vulnerability management?**
 
-With GitBOM, we propose a solution to both the completeness and the efficiency challenges facing other supply chain tools.
+[GitBOM](glossary/gitbom/) proposes a solution to both the completeness and the efficiency challenges facing other supply chain tools.
 
-- By correlating every piece of software with a verifiable and complete tree of all the "ingredients" that went into it, GitBOM enables the identification of software derived from sources known to contain vulnerabilities.
-- Unlike SBOMs, which contain metadata unrelated to the task of representing the dependency graph, GitBOM only includes the minimum information -- a "fingerprint" -- to enable efficient run-time scanning.
-- A GitBOM [artifact tree](/glossary/artifact_tree) can be cross-referenced against known vulnerabilities, regardless of the dependency depth and language.
+- By correlating every piece of software with a verifiable and complete tree of all the "ingredients" that went into it (source files, dependencies, object files, etc.), [GitBOM](glossary/gitbom/) enables the identification of software derived from sources known to contain vulnerabilities.
+- [GitBOM](glossary/gitbom/) only includes the minimum information -- a "fingerprint" -- of the dependency graph, to enable efficient run-time scanning for a known-vulnerable artifact
+- A [GitBOM](glossary/gitbom/) [artifact tree](/glossary/artifact_tree) can be cross-referenced against known vulnerabilities, regardless of the dependency depth or language.
 
-We believe this approach can work across all packaging formats, language ecosystems, and operating systems.
+**How does [GitBOM](glossary/gitbom/) work?**
```

Comment:
You might try doing this as a header (like # or ## or ###) instead of just bold, as it would make it linkable :)

Reply:
done :)